src/common/mfu_daos.c

```diff
@@ -1397,19 +1547,32 @@ int daos_setup(
             }
         }
 
+        /* TODO: pass daos_preserve bool here to daos_connect=true */
         create_cont = true;
+        /* do check that src is POSIX and since dst has a pool,
+         * then the dst *should* always be DFS, but this is not set
+         * on the destintaion until after daos_connect is called, and
+         * we should read the properties *before* the container is
+         * created. We do not have the destination container type
+         * here yet which is why extra check is used then passed
+         * to daos_connect */
+        if (da->daos_preserve) {
+            if (mfu_src_file->type == POSIX) {
+                preserve = true; 
+            }
+        }
```

Comment:
@dsikich I wonder if we DO want to allow `--daos-preserve` for DAOS->DAOS?
Something like:
```
if preserve:
    use preserve file for dst cont create
else:
    use src cont for dst cont create
```

Otherwise, I think we should probably error here, so the user knows that the `--daos-preserve` they passed in isn't going to be used